src/Employer/Employer.Web/Views/Part1/Employer/Employer.cshtml

```diff
@@ -1,21 +1,19 @@
-@using Microsoft.Extensions.Options;
-@using Esfa.Recruit.Shared.Web.Configuration;
 @using Esfa.Recruit.Employer.Web.ViewModels.Part1.Employer;
-@using Esfa.Recruit.Shared.Web.Mappers
 @model EmployerViewModel;
-@{ 
+@{
     ViewBag.Vpv = "/recruitment/employer/page-part1-employer";
-    ViewBag.ZenDeskLabel = "recruit-which-organisation-is-this-vacancy-for";
+    ViewBag.ZenDeskLabel = "recruit-which-organisation-is-this-advert-for";
```

Comment:
if we are changing the zendesk label, we need to make sure the article tag has also been changed